features/step_definitions/routability.js

```diff
@@ -29,8 +29,13 @@ module.exports = function () {
                                     result[direction].status.toString() : '';
                                 break;
                             case /^\d+s/.test(want):
+                                outputRow[direction] = result[direction].time ?
+                                    result[direction].time.toString()+'s' : '';
                                 break;
                             case /^\d+ km\/h/.test(want):
+                                outputRow[direction] = !isNaN(result[direction].speed) ?
+                                    result[direction].speed.toString()+' km/h' :
+                                    result[direction].speed.toString() || '';
```

Comment:
From skimming the failing tests, most are +- 1 km/h; but I also spotted more severe issues:

```
| (-) track | (-) asphalt | (-) 48s | 
| (+) track | (+) asphalt | (+) 60s | 
```
```
| (-) service | (-) driveway | (-) 15 km/h +-1 | (-) 15 km/h +-1 | 
| (+) service | (+) driveway | (+) 23 km/h | (+) 23 km/h | 
```

once you have a small epsilon of 1 or 2 km/h in the comparison you'll probably only see those handful of tests failing. And then we have to check what's going on there.